packages/regl-worldview/src/commands/Lines.js

```diff
@@ -463,14 +464,23 @@ const lines = (regl: any) => {
     }
   }
 
+  // Create a new render function based on rendering settings
+  // Memoization is required in order to prevent creating too many functions
+  // that use the same arguments, potentially leading to memory leaks.
+  const memoizedRender = memoize(
+    (props: { debug?: boolean, depth?: DepthState, blend?: BlendState }) => {
+      const { debug, depth = defaultReglDepth, blend = defaultReglBlend } = props;
+      if (debug) {
+        return regl({ depth: { enable: false } });
+      }
+      return regl({ depth, blend });
```

Comment:
We could make this `memoizedRender ` a bit simpler by moving the `debug` flag check out of this function and into `renderLine`. Then we could just memoize on `depth` and `blend` here:

```
const memoizedRender = memoize(
    ({ depth = defaultReglDepth, blend = defaultReglBlend }: { depth?: DepthState, blend?: BlendState }) => regl({ depth, blend }),
    (...args) => JSON.stringify(args)
  );
```

Reply:
Wouldn't that end up creating more `render` functions for debug mode only?